theano/gpuarray/tests/test_basic_ops.py

```diff
@@ -182,6 +182,24 @@ def test_transfer_cpu_gpu():
     assert numpy.all(fv == av)
 
 
+def test_transfer_gpu_gpu():
+    ctxs = list_contexts()
+    if len(ctxs) <= 1:
+        raise SkipError("Need more then 1 context to test multi-gpu transfer")
```

Comment:
You can still do a GpuToGpu in the same context.  It should do the equivalent of a copy.